inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/StreamSourceEntityMapper.java

```diff
@@ -41,6 +41,9 @@ public interface StreamSourceEntityMapper {
      */
     StreamSourceEntity selectForAgentTask(Integer id);
 
+    StreamSourceEntity selectExistsByTemplateIdAndIp(@Param("templateId") Integer templateId,
```

Comment:
1. Please add Javadoc for the method.
2. Maybe `selectOneByTemplatedIdAndAgentIp` is more readable.

Reply:
done.